tools/rocksdb-stats/src/main.rs

```diff
@@ -0,0 +1,85 @@
+use nearcore::{get_default_home, get_store_path};
+use serde::Serialize;
+use std::collections::HashMap;
+use std::process::Command;
+
+#[derive(Serialize, Default, Debug)]
+struct Data {
+    col: String,
+    entries: u64,
+    estimated_table_size: u64,
+    // Total size of all keys in bytes.
+    raw_key_size: u64,
+    // Total size of all values in bytes.
+    raw_value_size: u64,
+}
+
+impl Data {
+    pub fn merge(&mut self, other: &Data) {
+        self.entries += other.entries;
+        self.estimated_table_size += other.estimated_table_size;
+        self.raw_key_size += other.raw_key_size;
+        self.raw_value_size += other.raw_value_size;
+    }
+}
+
+fn parse_sst_file_dump(lines: &[&str]) -> Data {
+    let mut data = Data::default();
+    for line in lines {
+        let split_line: Vec<&str> = line.split(':').collect();
+        if split_line.len() < 2 {
+            continue;
+        }
+        let line = split_line[0].trim();
+        let value = split_line[1].trim();
+        eprintln!("{} {}", line, value);
+
+        match line {
+            "column family name" => data.col = value.to_string(),
```

Comment:
Would be nice to have an assert of some kind to ensure that this line does not appear in the output multiple times.

(Or a comment if the `sst` format guarantees that this line can only appear 0 or 1 times in the output)